docs/4_4_3_flexray.adoc

```diff
@@ -1,3 +1,1004 @@
-==== FlexRay
+==== FlexRay [[low-cut-flexray]]
+This chapter describes the <<low-cut-layered-standard-bus-protocol, Layered Standard Bus Protocol>> for FlexRay.
 
-#TODO#
+===== Overview [[low-cut-flexray-overview]]
+This specification refers primary to the data link layer specification of FlexRay communications systems defined by <<ISO-17458-2,  ISO 17458-2:2013(en)>>.
+Furthermore, essential elements from previous FlexRay specifications are covered so that these can also be mapped using this layered standard.
+
+To simulate FlexRay communication between Networked FMUs, FlexRay-specific operations are specified based on the <<low-cut-layered-standard-bus-protocol, Layered Standard Bus Protocol>>.
+Overall, the following operations and groups of operations exists:
+
+* Transmit, Cancel and Confirm: This operations are used to simulate a FlexRay frame transmission.
```

Comment:
Transmit, Cancel and Confirm: This (These) operations are used to simulate a FlexRay frame transmission.

Reply:
@kahramonj thanks I fixed it.